code/core/src/manager/components/TourGuide/TourTooltip.tsx

```diff
@@ -96,7 +111,7 @@ export const Tooltip: FC<TooltipProps> = ({
 }) => {
   useEffect(() => {
     const style = document.createElement('style');
-    style.id = '#sb-onboarding-arrow-style';
+    style.id = ONBOARDING_ARROW_STYLE_ID;
     style.innerHTML = `
       .__floater__arrow { container-type: size; }
       .__floater__arrow span { background: ${color.secondary}; }
```

Comment:
If these are global styles to bypass the styling API of `react-floater` (it would be nice to know why their styling APIs are inadequate, and why we chose to use the library at all.

The global styles can just be injected into the manager global CSS, there really is no need to keep injecting and removing a style-tag.